src/models/portfolio/schema.js

```diff
@@ -3,6 +3,10 @@ const Schema = mongoose.Schema;
 const ObjectId = Schema.Types.ObjectId;
 
 const schema = new Schema({
```

Comment:
so we have multiple users, and each user can have multiple portfolios, but also each portfolio has multiple ETFs, right?

Reply:
Correct